ideascube/serveradmin/catalog.py

```diff
@@ -567,7 +572,24 @@ def _load_catalog(self):
 
             # yaml.safe_load returns None for an empty file
             if catalog is not None:
-                self._catalog = catalog
+                if 'available' in catalog and 'installed' in catalog:
+                    # The cache on file hasn't been splitted yet
```

Comment:
`yet` here seems abiguous to me: yet in stack or yet in time? I know what you mean, but if someone else than you and me read this it may not understand. I'll comment something like "we are loading a catalog in a former format" or so.